fastlane/Fastfile

```diff
@@ -0,0 +1,179 @@
+    end 
+  end
+end
+
+desc "Build hybrid example"
+lane :build_hybrid_example do |options|
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("package.json")
+    Dir.chdir("..")
+  end  
+
+  sh("yarn example")
+  Dir.chdir("examples/purchaseTester/ios") do
+    sh("pod update PurchasesHybridCommon")
+  end
+  options[:previous_pod_version] = parse_pod_version
+end
+
+desc "Update iOS pod version during hybrid update"
+lane :update_ios_pod do |options|
+  if !File.file?("RNPurchases.podspec")
+    Dir.chdir("..")
+  end 
+  if options[:dry_run]
+    dry_run = true
+  end
+  if options[:version]
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  if options[:previous_pod_version]
+    previous_pod_version = options[:previous_pod_version]
+  else
+    UI.user_error!("Missing `previous_pod_version` argument")
+  end
+
+  if options[:previous_hybrid_version]
+    previous_hybrid_version = options[:previous_hybrid_version]
+  else
+    UI.user_error!("Missing `previous_hybrid_version` argument")
+  end
+
+  new_pod_version=parse_pod_version
+  
+  UI.message("ℹ️  Current Purchases version: #{previous_hybrid_version}")
+  UI.message("ℹ️  New Purchases version    : #{new_version_number}")
+  UI.message("ℹ️  Current Pod version      : #{previous_pod_version}")
+  UI.message("ℹ️  New Pod version          : #{new_pod_version}")
+  if dry_run
+    UI.message("ℹ️  Nothing more to do, dry_run: true")
+  else
+    UI.message("ℹ️  Current dir:#{sh("pwd")}")
+    Dir.chdir("scripts") do
+      sh("sed -i.bck \"s/download-purchases-framework.sh #{previous_pod_version}/download-purchases-framework.sh #{new_pod_version}/\" build.js")
+      sh("sed -i.bck \"s/download-purchases-common.sh #{previous_hybrid_version}/download-purchases-common.sh #{new_version_number}/\" build.js")
+    end
+  end
+end
+
+desc "Update Hybrid Common, Build PurchaseTester, and updates download scrips. Calls update_hybrid_common, then build_hybrid_example then update_ios_pod"
+lane :update_hybrid_build_sample_update_scripts do |options|
+  update_hybrid_common(options)
+  build_hybrid_example(options)
+  update_ios_pod(options)
+end
+
+###############################################################################
+# Helper functions 🤜🤛                                                      #
+###############################################################################
+
+def parse_previous_hybrid_version
+  return (sh "cat RNPurchases.podspec | grep \'\"PurchasesHybridCommon\", \' | awk \'{print($3)}\' | sed \"s/\'//g\"").strip
+end
+
+def increment_build_number(previous_version_number, new_version_number, file_path)
+  replace_in(previous_version_number, new_version_number, file_path)
+end
+
+def parse_pod_version
+  return sh("cat examples/purchaseTester/ios/Podfile.lock | grep \' Purchases (=\' | awk \'{print($4)}\' | sed \"s/)//g\"").strip
```

Comment:
perhaps these calls to `cat` can be wrapped nicely like in `replace_in`, so they're easier to use for folks who're not used to them?

Reply:
we could even move the previous versions to a separate file, like we did [here](https://github.com/RevenueCat/purchases-ios/blob/main/.version), to make the parsing less brittle

Reply:
Going to leave this for now like it is because I want to merge this ASAP, but I think you are right and we can do `.version` files. I was using the `Podfile.lock` in the example to figure out which version of Purchases the PurchasesHybridCommon pod was bringing. I think we should just remove `build.js` and force the usage of Cocoapods.